lib/rsolr/client.rb

```diff
@@ -63,6 +65,11 @@ def paginate page, per_page, path, opts = nil
     execute build_paginated_request(page, per_page, path, opts)
   end
   
+  # Sends a query to /select or the custom configured select handler.
+  def select opts={}
```

Comment:
Right, I think I see your point. I'll remove the select method and configuration from here and move that stuff in ActiveFedora. 

Would you rather have the update_path here passed to initialize, like it is now, or should I change that to be in the opts parameter of update instead?